src/tools/clippy/tests/ui/borrow_interior_mutable_const/auxiliary/helper.rs

```diff
@@ -6,11 +6,11 @@
 use std::sync::atomic::AtomicUsize;
 
 enum Private<T> {
-    ToBeUnfrozen(T),
-    Frozen(usize),
+    ToBeUnfrozen(#[allow(unused_tuple_struct_fields)] T),
```

Comment:
NIT: Usually we allow this lints with an inner attribute at the top of the file in Clippy. This will require some `stderr` update, which you can do with `x.py test src/tools/clippy --bless`.

Reply:
r=me after addressing this comment to conform to clippy's test style guide